vignettes/projecting_incidence.Rmd

```diff
@@ -0,0 +1,398 @@
+provided we can characterize the distribution of times between
+successive cases (serial interval), and the distribution of secondary cases
+produced by a single individual (offspring distribution). Such simulations can
+be achieved in _epichains_ with the `simulate_tree()` function and
+@pearson2020, and @abbott2020 illustrate its application to COVID-19.
+
+The purpose of this vignette is to use early data on COVID-19 in South Africa 
+[@marivate2020] to illustrate how _epichains_ can be used to project an 
+outbreak. 
+
+Let's load the required packages
+
+```{r packages, include=TRUE}
+library("epichains")
+library("dplyr")
+library("ggplot2")
+library("lubridate")
+```
+
+## Data
+
+Included in _epichains_ is a cleaned time series of the first 15 days of 
+the COVID-19 outbreak in South Africa. This can be loaded into 
+memory as follows: 
+```{r}
+data("covid19_sa", package = "epichains")
+```
+
+We will use the first $5$ observations for this demonstration. We will assume
+that all the cases in that subset are imported and did not infect each other. 
+
+Let us subset and view that aspect of the data.
+```{r}
+seed_cases <- covid19_sa[1:5, ]
+head(seed_cases)
+```
+
+## Setting up the inputs  
+
+We will now proceed to set up `simulate_tree()` for the simulations.
+
+### Onset times 
+
+`simulate_tree()` requires a vector of seeding times, `t0`, for each
+chain/individual/simulation.
+
+To get this, we will use the observation date of the index case as the
+reference and find the difference between the other observed dates and the reference. 
+```{r linelist_gen, message=FALSE}
+days_since_index <- as.integer(seed_cases$date - min(seed_cases$date))
+days_since_index
+```
+
+Using the vector of start times from the time series, we will then 
+create a corresponding seeding time for each individual, which we'll call
+`tf`.
+```{r}
+t0 <- rep(days_since_index, seed_cases$cases)
+t0
+```
+
+### Generation time
+
+In epidemiology, the generation time (also called the generation interval) is
+the duration between successive infectious events in a chain of transmission.
+Similarly, the serial interval is the duration between observed symptom onset
+times between successive cases in a transmission chain. The generation
+interval is often hard to observe because exact times of infection are hard to
+measure hence, the serial interval is often used instead. Here, we
+use the serial interval and interpret the simulated case data to represent
+symptom onset.
+
+In this example, we will assume based on COVID-19 literature that the 
+serial interval, S, is log-normal distributed with parameters, 
+$\mu = 4.7$ and $\sigma = 2.9$ [@pearson2020]. The log-normal distribution is
+commonly used in epidemiology to characterise quantities such as the serial
+interval because it has a large variance and can only be positive-valued
+[@nishiura2007; @limpert2001].
+
+Note that when the distribution is described this way, it means $\mu$ and
```

Comment:
This was previously done with {epiparameter} but was later removed because of the consideration around delaying the use of {epiparameter} until the CRAN release. 

I am also wondering if this explanation is too much for the user, considering this conversion is often done in the literature with little explanation. The reasoning here is that a vignette should not only communicate package functionality but in the case of demonstrating use cases, it should also take advantage of teachable moments. 

Thoughts?